specs/paths/master/Overview.html

```diff
@@ -1368,10 +1368,10 @@ <h3 id="InterfaceSVGPathSegment">Interface SVGPathSegment</h3>
 The <a>SVGPathSegment</a> interface is a base interface that corresponds to a
 single command within a path data specification.
 
-<pre class="idl">[NoInterfaceObject]
+<pre class="idl">
 interface <b>SVGPathSegment</b> {
```

Comment:
Looking at how this is used, maybe this can even be a dictionary.

Reply:
I think you're right, that would be fine in this case, and would avoid the need to make `values` a `FrozenArray<float>` as one would need to do to keep this an interface.